src/OmniSharp.Cake/Services/RequestHandlers/Navigation/FindSymbolsHandler.cs

```diff
@@ -16,6 +18,17 @@ public FindSymbolsHandler(OmniSharpWorkspace workspace)
         {
         }
 
+        public override Task<QuickFixResponse> Handle(FindSymbolsRequest request)
+        {
+            Func<string, bool> isMatch =
+                candidate => request != null
+                ? candidate.IsValidCompletionFor(request.Filter)
+                : true;
+
+            return  Workspace.CurrentSolution.FindSymbols(isMatch,
+                p => p.Name.EndsWith(".cake", StringComparison.OrdinalIgnoreCase));
```

Comment:
Could we reuse the extension as defined in the project system?

Reply:
Maybe the project system and this handler should point to the same constant?